lib/main.js

```diff
@@ -481,4 +481,29 @@ if (options.interactiveTests) {
       })
     })
   }
+
+  ava.test.skip('ResinOS image: Check host OS fingerprint file on resin-root partition', async (test) => {
```

Comment:
Any reason to skip this test?

Reply:
@jviotti You have to run md5sum for every file from the fingerprint file. This is not feasible for manual testing.

Reply:
In that case, this test is not runnable, so can we create a GH issue instead, and handle it directly with code?